src/main/java/com/me4502/modularframework/module/ModuleWrapper.java

```diff
@@ -144,6 +159,26 @@ public String getName() {
         return moduleClassName;
     }
 
+    public Optional<String> getVersion() {
```

Comment:
This will only be empty in exceptional circumstances. I'd just have it return null if an exception is thrown rather than using optional.